packages/fether-electron/src/main/app/methods/setupRequestListeners.js

```diff
@@ -8,6 +8,30 @@ import electron from 'electron';
 import messages from '../messages';
 const { ipcMain, session } = electron;
 
+console.log(
+  'Configuring Content-Security-Policy for environment: ',
+  process.env.NODE_ENV
+);
+// Note: `worker-src blob:` required for the webcam
+const CSP =
+  process.env.NODE_ENV === 'development'
+    ? `
+    default-src 'self';
+    script-src 'self' file: http: blob: 'unsafe-inline' 'unsafe-eval';
+    connect-src 'self' file: http: https: ws: wss:;
+    img-src 'self' 'unsafe-inline' file: data: blob: http: https:;
+    style-src 'self' 'unsafe-inline' file: blob:;
+    object-src 'none';
```

Comment:
Thanks. Yes, we're creating a `new window.Blob` to backup accounts. 

So it looks like using object-src isn't necessary since we're not doing any of this https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/object-src#Examples